tests/reference/asr-bindc_02-5092d8e.stderr

```diff
@@ -1,8 +1,2 @@
-semantic error: Type mismatch in procedure call; the types must be compatible
- --> tests/errors/bindc_02.py:9:24
-  |
-9 |     s: Struct = Struct(int32obj)
-  |                        ^^^^^^^^ type mismatch (passed argument type is i32 but required type is CPtr)
-  |
-5 |     cptr_member: CPtr
-  |                  ^^^^ type mismatch (passed argument type is i32 but required type is CPtr)
+'lpython' is not recognized as an internal or external command,
+operable program or batch file.
```

Comment:
@khairalanam please see above. I think the tests are not properly updated.

Reply:
What is the expected output of these reference tests? I couldn't find any errors in the terminal.

![image](https://user-images.githubusercontent.com/84736189/228783147-d97286c9-bde0-46c9-bf6b-6c85dbdcedcb.png)

Reply:
>What is the expected output of these reference tests?

The expected output of reference tests is as stored in the main branch.

> I couldn't find any errors in the terminal.

I think its a bug in the `run_tests.py -u` that it runs (updates tests) even if it cannot find `lpython`.

Reply:
Its a Windows specific problem. Try to ask about it on Zulip.